docs/source/tutorials/structure.rst

```diff
@@ -124,7 +127,7 @@ In this package every :code:`Model` object that will be used in the project are
 message
 ^^^^^^^
 
-In this package all the messages that will be displayed in the app are gathered in json dictionaries. More information in this tutorial
+In this package all the messages that will be displayed in the app are gathered in json dictionaries. More information in this tutorial.
```

Comment:
What do you mean with `this tutorial`? This structure page? or are we missing a link?

Reply:
missing link

Reply:
In fact all the links were missing, normally it's corrected